### ISSUE TITLE
Imported class throws not found fatal error

### ISSUE THREAD

**Issue by cvetan** [sandhje/vscode-phpmd#44]
phpmd throws unreasonable error in vscode console for the imported class. For example, there is sample controller from Laravel application:

```
<?php

namespace App\Http\Controllers;

use Illuminate\Foundation\Bus\DispatchesJobs;
use Illuminate\Routing\Controller as BaseController;
use Illuminate\Foundation\Validation\ValidatesRequests;
use Illuminate\Foundation\Auth\Access\AuthorizesRequests;

class Controller extends BaseController
{
    use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
}
```
It throws this error:
```
[Error - 1:49:35 PM] phpmd: PHP Fatal error:  Class 'Illuminate\Routing\Controller' not found in /home/cvetan/Code/cvetansimsic/app/Http/Controllers/Controller.php on line 10
```

**Comment by cvetan** [comment]
How do I do that? :)

**Comment by sandhje** [comment]
In File > Preferences > Settings, Click on Extensions > PHP Mess Detector > Enable verbose logging. Then open the output window (View > output) and select PHP Mess Detector in the dropdown. Then trigger the error you are reporting and copy the output here.

**Comment by cvetan** [comment]
Looks like not setting phpmd path, made extension missbehave. When I looked for this extension specific settings, I've noticed that phpmd path is not set. 

When I've set it, this problem was not persistent. But I thought if not setting phpmd path, it will use default system path. And it says if the php path is set, you can use buildint phpmd executable.

Anyways, I think this is solved then. :)
